<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUserManagerDlg</class>
 <widget class="QDialog" name="QUserManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Consolas&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>当前登录用户：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_UserName">
       <property name="text">
        <string>UserName</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="m_pbModifyUser">
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="m_pbDelUser">
     <property name="text">
      <string>删除用户</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="m_pbAddUser">
     <property name="text">
      <string>添加用户</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_Status">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
